.github/BOTMETA.yml

```diff
@@ -1121,7 +1123,7 @@ macros:
   team_networking: Qalthos ganeshrn rcarrillocruz trishnaguha gdpak justjais NilashishC
   team_nso: cmoberg cnasten tbjurman
   team_nxos: mikewiebe rahushen rcarrillocruz trishnaguha tstoner
-  team_onyx: samerd anasb
```

Comment:
this should be `anasbadaha` instead of `anasb` - see #47552

Reply:
Alright, then the module is wrong too. I will fix.

Reply:
Fixed.